packages/connectivity/lib/connectivity.dart

```diff
@@ -56,6 +56,17 @@ class Connectivity {
     if (wifiName == '<unknown ssid>') wifiName = null;
     return wifiName;
   }
+
+  /// Obtains the wifi ip (IP) of the connected network
+  ///
+  /// Please note that it DOESN'T WORK on emulators (returns null).
+  Future<String> getWifiIP() async {
+    String wifiIP = await _methodChannel.invokeMethod('wifiIPAddress');
+    // as Android might return <unknown ssid>, uniforming result
+    // our iOS implementation will return null
+    if (wifiIP == '<unknown ip>') wifiIP = null;
```

Comment:
I think you don't actually need this comment and `if wifiIP == '<unknown ip>'`, right?

In `getWifiName` that's required because that implementation *actually* returns '<unknown ssdi>' instead of null.

Reply:
Done!

Reply:
But have you kept the variable? It's not needed anymore, isn't it?